integration/build.test.ts

```diff
@@ -0,0 +1,62 @@
+import {PrimerStyleDictionary} from '../src/PrimerStyleDictionary'
+import fs from 'fs'
+
+describe('PrimerStyleDictionary', () => {
+  const basePath = `./integration`
+  const extendedSD = PrimerStyleDictionary.extend({
+    source: [`${basePath}/tokens/**/*.json5`],
+    platforms: {
+      css: {
+        prefix: 'PREFIX',
+        transformGroup: 'css',
+        buildPath: `${basePath}/build/css/`,
+        files: [
+          {
+            options: {
+              showFileHeader: false,
+            },
+            destination: 'variables.css',
+            format: 'css/variables',
+          },
+        ],
+      },
+      advancedCss: {
+        prefix: 'PREFIX',
+        transformGroup: 'css',
+        buildPath: `${basePath}/build/css/`,
+        files: [
+          {
+            options: {
+              showFileHeader: false,
+            },
+            destination: 'advanced.css',
+            format: 'css/advanced',
+          },
+        ],
+      },
+    },
+  })
+
+  extendedSD.cleanAllPlatforms()
+  extendedSD.buildAllPlatforms()
+
+  it('it should transform with css/variables format', () => {
+    const output = fs.readFileSync(`${basePath}/build/css/variables.css`, 'utf8')
+    const expectedOutput = `:root {
+  --prefix-base-color-blue-500: #2C29FF; /* The primary color for interactive elements. */
```

Comment:
qq... should this test instead assert that the prefix isn't applied to base tokens and only functional ones?

Reply:
Why? This is not the expected behavior for this config.

Reply:
Based on https://github.com/primer/primitives/pull/1040/files#r1740667105 I think I got my answer. The opinionated rules are applied in a different stage in the build process, so this isn't representative of an end-to-end test.

Reply:
Yes, I will add tests for this as well. Those are the baselines. :D